pkg/api/networkservice/connection.proto

```diff
@@ -25,53 +25,54 @@ import "ptypes/timestamp/timestamp.proto";
 import "connectioncontext.proto";
```

Comment:
Why does this file have only whitespace changes?

Reply:
Remove whitespace changes. Thanks.